assets/src/components/account/UserProfile.tsx

```diff
@@ -83,6 +88,9 @@ class UserProfile extends React.Component<Props, State> {
   handleChangeProfilePhotoUrl = (e: any) => {
     this.setState({profilePhotoUrl: e.target.value});
   };
+  handleChangeEmail = (e:any) => {
```

Comment:
Can we do something like `handleChangeEmail = (e: ChangeEvent<HTMLInputElement>) => {`?

Reply:
(+1 that adding better types here would be nice)